source/Calamari.Tests/Fixtures/PowerShell/PowerShellFixture.cs

```diff
@@ -28,9 +24,10 @@ public void Teardown()
         }
 
         [Test]
+        [Platform]
         [Category(TestCategory.CompatibleOS.Windows)]
-        [TestCase("2", "PSVersion                      2.0")]
-        [TestCase("2.0", "PSVersion                      2.0")]
+        [TestCase("2", "PSVersion                      2.0", IncludePlatform = "Win2008Server,Win2008ServerR2,Win2012Server,Win2012ServerR2")]
```

Comment:
Bit scared about that massive space in the `expectedLogMessage`, but I think I'll go 🙈.

The expected platforms should probably include windows 10 (for local dev) and Windows 2016.

Reply:
Added Windows 10. The nunit implementation to detect for Server 2016 will also match Server 2019 so we need to roll our own. My thinking was that for an obscure feature with such a low risk of failing on just one server version it was not worth rolling our own.

Reply:
Sounds reasonable. Wanna add a comment to that effect?